docs/source/user-guide/general/customizations/plot_create_a_custom_check.py

```diff
@@ -0,0 +1,17 @@
+"""
+=====================
+Create a Custom Check
+=====================
+
+It is possible to extend deepchecks by implementing custom checks. This
+enables you to have your own logic of metrics or validation, or even just
+to display your own graph using deepchecks' suite.
+
+While some aspects for writing a check are similar when writing checks for the
+``deepchecks.tabular`` ``and deepchecks.vision`` subpackages, there are major differences
+that warrant a separate guide for each:
```

Comment:
IMO this paragraph is more confusing than helpful, we should just say something like: here is the guide for tabular (tabular link) and here is the guide for vision (vision link)